interface/wx/statusbr.h

```diff
@@ -248,6 +248,22 @@ class wxStatusBar : public wxControl
     */
     virtual void SetFieldsCount(int number = 1, const int* widths = nullptr);
 
+    /**
+        Add a control (child of the wxStatusBar) to be shown at the specified
+        field position in the status bar.
+
+        @param i
+            The field index where the control will be shown.
+        @param win
+            The control in question. Normally a child of the wxStatusBar itself.
+
+        @note You must delete the control to remove it from the status bar, as
+            simply passing @NULL will not do that.
+
+        @since 3.2.5
```

Comment:
This can't be easily backported (it adds new fields), so I'd use
```suggestion
        @since 3.3.0
```